### ISSUE TITLE
Mismatch in namespaces of Context, Service and UI ontologies in web

### ISSUE THREAD

**Issue by Alfiva** [universAAL/ontology#4]
This is actually related to the code of Context, Service and UI bus, but since it has to do with their ontologies, I report it here:

If you visit http://ontology.universaal.org/ to check for our published ontologies, you will find that things like ContextProvider are found in http://ontology.universaal.org/ContextBus.owl rather than in http://ontology.universaal.org/Context.owl as their namespace indicates (e.g. http://ontology.universaal.org/Context.owl#ContextProvider ; this link will not be found). The same happens for Service and UI ontologies.

In other words, the OWL (and TTL) file names in the web should drop the "Bus" suffix, so that namespaces are correct and resources can be located e.g. with a web browser.
